apiary.apib

```diff
@@ -3713,6 +3713,25 @@ Send a finalized work order to the customer.
 
 + Response 204
 
+### workOrders.download [POST /workOrders.download]
+
+Download a work order in a specific format.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 200 (application/json;charset=utf-8)
```

Comment:
Didn't we remove the `charset` from the responses/requests?

Reply:
Copied blindly from the proposal..  🤦‍♂️ 
Should we do this for all responses?

Reply:
It's like that for master already. 
So I guess we should change that to all requests/responses in `work-orders` branch?